src/slapper/Main.php

```diff
@@ -79,7 +89,9 @@ class Main extends PluginBase implements Listener {
 		"Horse", "Donkey", "Mule", "SkeletonHorse",
 		"ZombieHorse", "Witch", "Rabbit", "Stray",
 		"Husk", "WitherSkeleton", "IronGolem", "Snowman",
-		"MagmaCube", "Squid"
+		"MagmaCube", "Squid", "ElderGuardian", "Endermite",
+		"Evoker", "Guardian", "PolarBear", "Shulker",
+		"Vex", "Vindicator", "Wither", "Llama"
```

Comment:
Why not move these into their respective classes? Then just have a wrapper for Entity::registerEntity() that will register the names into here before registering the entity. You'd have to make this non-const though.